src/main/java/org/mule/extension/db/api/config/DbPoolingProfile.java

```diff
@@ -82,6 +89,11 @@
   @Expression(NOT_SUPPORTED)
   private int maxIdleTime;
 
+  @Parameter
+  @Optional
+  @Placement(tab = ADVANCED_TAB)
+  private Map<String, Object> additionalProperties = emptyMap();
```

Comment:
Add a summary of this new parameter:
https://docs.mulesoft.com/mule-sdk/1.1/define-parameters#layout